source/help/econ/index.md

```diff
@@ -12,11 +12,11 @@ The [moderators for the economics archive are listed here](https://arxiv.org/mod
 
 The editorial committee members serve as consultants to Cornell University and to the [arXiv Editorial Advisory Committee](../../about/people/editorial_advisory_council.md). All arXiv policy decisions are ultimately made by Cornell University.
 
-- Joerg Stoye, Cornell University (chair)
-- Martin Weidner, University of Oxford (vice chair)
+- Marc Henry, The Pennsylvania State University (Chair)
+- Martin Weidner, University of Oxford (Vice Chair)
```

Comment:
In keeping with the format on this and other related pages, "chair" and "vice chair" should be lower case.